proto/spire/next/types/bundle.proto

```diff
@@ -16,6 +16,10 @@ message Bundle {
     // A hint on how often the bundle should be refreshed from the bundle
     // provider, in seconds. Can be zero (meaning no hint available).
     int64 refresh_hint = 4;
+
+    // The sequence number of the bundle. SPIRE does not use sequence numbers
+    // and will treat this number as an opaque value.
```

Comment:
It's probably a good idea to avoid codifying SPIRE's current behavior in text that might be overlooked if/when it changes

Reply:
:+1: